components/linear/linear.app.mjs

```diff
@@ -11,5 +11,43 @@ export default {
         ...options,
       };
     },
+    async getOwnUserInfo() {
+      const {
+        avatarUrl, displayName,
+      } = await this.client().viewer;
+      return {
+        avatarUrl,
+        displayName,
+      };
+    },
+  },
+  propDefinitions: {
+    ...linearApp.propDefinitions,
+    createAs: {
+      type: "string",
+      label: "Create As User/App",
+      description: `Select who will appear as creating the issue on its history.
```

Comment:
"Specify to whom who you'd like to attribute the issue creation (the signed-in user vs a custom username)."

Reply:
Applied this change.